src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java

```diff
@@ -124,6 +121,7 @@ public void download(final String filePath) throws IOException {
     /**
      * Retrieves a  token from the endpoint from a username and password.
      *
+     * @throws IOException when introduced an invalid path

```

Comment:
You forgot the point at the end in this java doc

Reply:
ok dani, i will correct it